src/display.js

```diff
@@ -0,0 +1,57 @@
+import {
+  getTodoList, markCompleted, removeItem, editItem,
+} from './todoList.js';
+
+const listContainer = document.querySelector('.list');
+const displayToDoList = (todoList) => {
+  listContainer.innerHTML = '';
+  todoList.forEach((item, index) => {
+    const listItem = document.createElement('div');
+    listItem.className = 'list-properties';
+    listItem.innerHTML = `
+        <div class="both-sides">
+          <span class="left-items">
+            <input type="checkbox" ${item.completed ? 'checked' : ''} data-index="${index}">
+            <input type="text" value="${item.description}" class="input2">
+          </span>
+          <span class="right-items">
+            <div class="edit-item" data-index="${index}"></div>
+            <div class="remove-item" data-index="${index}"><i class="fa-solid fa-trash"></i></div>
+          </span>
+        </div>
+        <hr class="line-1">
+      `;
+    const checkBox = listItem.querySelector('input[type="checkbox"]');
+    checkBox.addEventListener('change', (event) => {
+      const index = event.target.getAttribute('data-index');
+      markCompleted(index);
+      displayToDoList(getTodoList());
+    });
+    const description = listItem.querySelector('input[type="text"]');
```

Comment:
- While a user is clicking on the checkbox a line-through style should be applied to the text input field. Try to add a class for it using CSS.

- When the user is checking the checkbox it is not possible to uncheck it again. Try to fix it so a user could check and uncheck a checkbox. (While unchecking the line-through style should be removed from the text input field.